pkg/evaluator/builtin.go

```diff
@@ -138,9 +147,19 @@ func newBuiltins(rt Runtime) builtins {
 		"input":   {Name: "input", Params: inputParams},
 		"animate": {Name: "animate", Params: numParam},
 	}
-	globals := map[string]*parser.Var{
-		"err":    {Name: "err", T: parser.BOOL_TYPE},
-		"errmsg": {Name: "errmsg", T: parser.STRING_TYPE},
+	globals := map[string]global{
+		"err": {
+			val:       &boolVal{V: false},
+			parserVar: &parser.Var{Name: "err", T: parser.BOOL_TYPE},
```

Comment:
nit: can you swap these two over (and below). Then it matches the order declared in the struct but also doesn't seem backwards to me: var then value is the ordering I'd expect.

Reply:
of course, agreed. fixed.